From 02cff123e9fae94ff9414819c3a0f0ae06b511ee Mon Sep 17 00:00:00 2001
From: Lawrence Mok <lawrence.mok@amlogic.com>
Date: Fri, 2 Jan 2015 16:08:14 -0800
Subject: [PATCH 5286/5965] meson8*: make savedefconfig

use 'make savedefconfig' to generate defconfigs

some configs look removed, but are actually default to enabled,
so they will be enabled when you make the defconfig

Change-Id: I5182a7679bfa52da0e7f17a59ecdc38da377907c
---
 arch/arm/configs/meson8_defconfig      | 39 +++++++++-----------------
 arch/arm/configs/meson8_tee_defconfig  | 37 +++++++++---------------
 arch/arm/configs/meson8b_defconfig     | 34 +++++++++-------------
 arch/arm/configs/meson8b_tee_defconfig | 32 ++++++++-------------
 4 files changed, 52 insertions(+), 90 deletions(-)
 mode change 100755 => 100644 arch/arm/configs/meson8_defconfig
 mode change 100755 => 100644 arch/arm/configs/meson8_tee_defconfig
 mode change 100755 => 100644 arch/arm/configs/meson8b_defconfig
 mode change 100755 => 100644 arch/arm/configs/meson8b_tee_defconfig

diff --git a/arch/arm/configs/meson8_defconfig b/arch/arm/configs/meson8_defconfig
old mode 100755
new mode 100644
index 7821ce19b7a8..14813d61e95c
--- a/arch/arm/configs/meson8_defconfig
+++ b/arch/arm/configs/meson8_defconfig
@@ -208,38 +208,22 @@ CONFIG_LIGHT_ISL29023=y
 CONFIG_LIGHT_STK220X=y
 CONFIG_LIGHT_LTR558=y
 CONFIG_AML_GPIO_KEY=y
-CONFIG_MESON_CS_DCDC_REGULATOR=y
 CONFIG_AML_PMU_ALGORITHM_SUPPORT=m
+CONFIG_MESON_CS_DCDC_REGULATOR=y
 CONFIG_AML_PMU=y
 CONFIG_AML1216=y
 CONFIG_AML1218=y
 CONFIG_RICOH_PMU=y
 CONFIG_SECURITYKEY=y
 CONFIG_EFUSE=y
-CONFIG_AMLOGIC_USB=y
 CONFIG_AML_PLATFORM_THERMAL=y
-CONFIG_THERMAL=y
-CONFIG_AML_VIRTUAL_THERMAL=y
-CONFIG_CPU_THERMAL=y
-CONFIG_GPU_THERMAL=y
-CONFIG_CPUCORE_THERMAL=y
-CONFIG_GPUCORE_THERMAL=y
+CONFIG_AMLOGIC_USB=y
 CONFIG_MMC_AML=y
 CONFIG_AMLOGIC_SPI_NOR=y
 CONFIG_AML_NEXT_GEN_NAND=y
 CONFIG_AML_NAND_KEY=y
 # CONFIG_AM_NAND is not set
-#new maclogic
-CONFIG_NET_VENDOR_STMICRO=y
-CONFIG_STMMAC_ETH=y
-CONFIG_STMMAC_PLATFORM=y
-CONFIG_DWMAC_MESON=y
-CONFIG_STMMAC_DEBUG_FS=y
-CONFIG_STMMAC_DA=y
-CONFIG_NET_VENDOR_WIZNET=y
 CONFIG_AM_PHY=y
-#new maclogic
-CONFIG_AM_ETHERNET=y
 CONFIG_AML_PHY=y
 CONFIG_AML_LAN8720=y
 CONFIG_AML_IP101_PHY=y
@@ -254,13 +238,11 @@ CONFIG_RTL8189ES=m
 CONFIG_BT_DEVICE=y
 CONFIG_BT_RTKBTUSB=m
 CONFIG_VSYNC_RDMA=y
-CONFIG_AM_VIDEO=y
+CONFIG_TVIN_VIUIN=y
+CONFIG_AM_VIDEO2=y
 CONFIG_AM_ENCODER=y
 CONFIG_AM_JPEG_ENCODER=y
 CONFIG_AM_TV_OUTPUT2=y
-CONFIG_AM_VIDEO2=y
-CONFIG_TVIN_VIUIN=y
-CONFIG_TVIN_VDIN=y
 CONFIG_FB_AM=y
 CONFIG_FB_OSD2_CURSOR=y
 CONFIG_FB_OSD2_DEFAULT_WIDTH=32
@@ -321,6 +303,10 @@ CONFIG_DM_UEVENT=y
 CONFIG_DM_VERITY=y
 CONFIG_NETDEVICES=y
 CONFIG_TUN=y
+CONFIG_STMMAC_ETH=y
+CONFIG_DWMAC_MESON=y
+CONFIG_STMMAC_DEBUG_FS=y
+CONFIG_STMMAC_DA=y
 CONFIG_PPP=y
 CONFIG_PPP_BSDCOMP=y
 CONFIG_PPP_DEFLATE=y
@@ -343,6 +329,11 @@ CONFIG_INPUT_UINPUT=y
 # CONFIG_LEGACY_PTYS is not set
 CONFIG_HW_RANDOM=y
 CONFIG_GPIO_SYSFS=y
+CONFIG_THERMAL=y
+CONFIG_CPU_THERMAL=y
+CONFIG_CPUCORE_THERMAL=y
+CONFIG_GPU_THERMAL=y
+CONFIG_GPUCORE_THERMAL=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_MEDIA_USB_SUPPORT=y
 CONFIG_USB_VIDEO_CLASS=y
@@ -396,6 +387,7 @@ CONFIG_UDF_FS=y
 CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
 CONFIG_EXFAT_FS=y
+CONFIG_NTFS_FS=y
 CONFIG_TMPFS=y
 CONFIG_HFS_FS=y
 CONFIG_HFSPLUS_FS=y
@@ -406,7 +398,6 @@ CONFIG_PSTORE_RAM=y
 CONFIG_CIFS=y
 CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ISO8859_1=y
-CONFIG_NLS_UTF8=y
 CONFIG_PRINTK_TIME=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_READABLE_ASM=y
@@ -425,5 +416,3 @@ CONFIG_CRYPTO_PCBC=y
 CONFIG_CRYPTO_TWOFISH=y
 CONFIG_CRC_T10DIF=y
 CONFIG_CRC7=y
-CONFIG_NTFS_FS=y
-
diff --git a/arch/arm/configs/meson8_tee_defconfig b/arch/arm/configs/meson8_tee_defconfig
old mode 100755
new mode 100644
index ee6efe6972ee..32bcfc73ab68
--- a/arch/arm/configs/meson8_tee_defconfig
+++ b/arch/arm/configs/meson8_tee_defconfig
@@ -209,8 +209,8 @@ CONFIG_LIGHT_ISL29023=y
 CONFIG_LIGHT_STK220X=y
 CONFIG_LIGHT_LTR558=y
 CONFIG_AML_GPIO_KEY=y
-CONFIG_MESON_CS_DCDC_REGULATOR=y
 CONFIG_AML_PMU_ALGORITHM_SUPPORT=m
+CONFIG_MESON_CS_DCDC_REGULATOR=y
 CONFIG_AML_PMU=y
 CONFIG_AML1216=y
 CONFIG_AML1218=y
@@ -218,29 +218,13 @@ CONFIG_RICOH_PMU=y
 CONFIG_SECURITYKEY=y
 CONFIG_EFUSE=y
 CONFIG_AML_PLATFORM_THERMAL=y
-CONFIG_AML_VIRTUAL_THERMAL=y
-CONFIG_THERMAL=y
-CONFIG_CPU_THERMAL=y
-CONFIG_GPU_THERMAL=y
-CONFIG_CPUCORE_THERMAL=y
-CONFIG_GPUCORE_THERMAL=y
 CONFIG_AMLOGIC_USB=y
 CONFIG_MMC_AML=y
 CONFIG_AMLOGIC_SPI_NOR=y
 CONFIG_AML_NEXT_GEN_NAND=y
 CONFIG_AML_NAND_KEY=y
 # CONFIG_AM_NAND is not set
-#new maclogic
-CONFIG_NET_VENDOR_STMICRO=y
-CONFIG_STMMAC_ETH=y
-CONFIG_STMMAC_PLATFORM=y
-CONFIG_DWMAC_MESON=y
-CONFIG_STMMAC_DEBUG_FS=y
-CONFIG_STMMAC_DA=y
-CONFIG_NET_VENDOR_WIZNET=y
 CONFIG_AM_PHY=y
-#new maclogic
-CONFIG_AM_ETHERNET=y
 CONFIG_AML_PHY=y
 CONFIG_AML_LAN8720=y
 CONFIG_AML_IP101_PHY=y
@@ -255,13 +239,11 @@ CONFIG_RTL8189ES=m
 CONFIG_BT_DEVICE=y
 CONFIG_BT_RTKBTUSB=m
 CONFIG_VSYNC_RDMA=y
-CONFIG_AM_VIDEO=y
+CONFIG_TVIN_VIUIN=y
+CONFIG_AM_VIDEO2=y
 CONFIG_AM_ENCODER=y
 CONFIG_AM_JPEG_ENCODER=y
 CONFIG_AM_TV_OUTPUT2=y
-CONFIG_AM_VIDEO2=y
-CONFIG_TVIN_VIUIN=y
-CONFIG_TVIN_VDIN=y
 CONFIG_FB_AM=y
 CONFIG_FB_OSD2_CURSOR=y
 CONFIG_FB_OSD2_DEFAULT_WIDTH=32
@@ -322,6 +304,10 @@ CONFIG_DM_UEVENT=y
 CONFIG_DM_VERITY=y
 CONFIG_NETDEVICES=y
 CONFIG_TUN=y
+CONFIG_STMMAC_ETH=y
+CONFIG_DWMAC_MESON=y
+CONFIG_STMMAC_DEBUG_FS=y
+CONFIG_STMMAC_DA=y
 CONFIG_PPP=y
 CONFIG_PPP_BSDCOMP=y
 CONFIG_PPP_DEFLATE=y
@@ -344,6 +330,11 @@ CONFIG_INPUT_UINPUT=y
 # CONFIG_LEGACY_PTYS is not set
 CONFIG_HW_RANDOM=y
 CONFIG_GPIO_SYSFS=y
+CONFIG_THERMAL=y
+CONFIG_CPU_THERMAL=y
+CONFIG_CPUCORE_THERMAL=y
+CONFIG_GPU_THERMAL=y
+CONFIG_GPUCORE_THERMAL=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_MEDIA_USB_SUPPORT=y
 CONFIG_USB_VIDEO_CLASS=y
@@ -398,6 +389,7 @@ CONFIG_UDF_FS=y
 CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
 CONFIG_EXFAT_FS=y
+CONFIG_NTFS_FS=y
 CONFIG_TMPFS=y
 CONFIG_HFS_FS=y
 CONFIG_HFSPLUS_FS=y
@@ -408,7 +400,6 @@ CONFIG_PSTORE_RAM=y
 CONFIG_CIFS=y
 CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ISO8859_1=y
-CONFIG_NLS_UTF8=y
 CONFIG_PRINTK_TIME=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_READABLE_ASM=y
@@ -427,5 +418,3 @@ CONFIG_CRYPTO_PCBC=y
 CONFIG_CRYPTO_TWOFISH=y
 CONFIG_CRC_T10DIF=y
 CONFIG_CRC7=y
-CONFIG_NTFS_FS=y
-
diff --git a/arch/arm/configs/meson8b_defconfig b/arch/arm/configs/meson8b_defconfig
old mode 100755
new mode 100644
index 444db92f3c51..3a00ff82d977
--- a/arch/arm/configs/meson8b_defconfig
+++ b/arch/arm/configs/meson8b_defconfig
@@ -211,30 +211,15 @@ CONFIG_AML1218=y
 CONFIG_RICOH_PMU=y
 CONFIG_SECURITYKEY=y
 CONFIG_EFUSE=y
-CONFIG_AMLOGIC_USB=y
 CONFIG_AML_PLATFORM_THERMAL=y
-CONFIG_AML_VIRTUAL_THERMAL=y
-CONFIG_THERMAL=y
-CONFIG_CPU_THERMAL=y
-CONFIG_GPU_THERMAL=y
-CONFIG_CPUCORE_THERMAL=y
-CONFIG_GPUCORE_THERMAL=y
+CONFIG_AMLOGIC_USB=y
 CONFIG_MMC_AML=y
 CONFIG_AMLOGIC_SPI_NOR=y
 CONFIG_AML_NEXT_GEN_NAND=y
 CONFIG_AML_NAND_KEY=y
 # CONFIG_AM_NAND is not set
-#new maclogic
-CONFIG_NET_VENDOR_STMICRO=y
-CONFIG_STMMAC_ETH=y
-CONFIG_STMMAC_PLATFORM=y
-CONFIG_DWMAC_MESON=y
-CONFIG_STMMAC_DEBUG_FS=y
-CONFIG_STMMAC_DA=y
-CONFIG_NET_VENDOR_WIZNET=y
-CONFIG_AM_PHY=y
-#new maclogic
 CONFIG_AM_ETHERNET=y
+CONFIG_AM_PHY=y
 CONFIG_AML_PHY=y
 CONFIG_AML_LAN8720=y
 CONFIG_AML_IP101_PHY=y
@@ -242,8 +227,8 @@ CONFIG_AML_KSZ8091=y
 CONFIG_AML_RTL8211F=y
 CONFIG_AM_WIFI=y
 CONFIG_AM_WIFI_SD_MMC=y
-CONFIG_AM_WIFI_USB=y
 CONFIG_DHD_USE_STATIC_BUF=y
+CONFIG_AM_WIFI_USB=y
 CONFIG_SDIO_DHD_CDC_WIFI_40181_MODULE=m
 CONFIG_RTL8188EU=m
 CONFIG_RTL8189ES=m
@@ -296,6 +281,10 @@ CONFIG_DM_CRYPT=y
 CONFIG_DM_UEVENT=y
 CONFIG_NETDEVICES=y
 CONFIG_TUN=y
+CONFIG_STMMAC_ETH=y
+CONFIG_DWMAC_MESON=y
+CONFIG_STMMAC_DEBUG_FS=y
+CONFIG_STMMAC_DA=y
 CONFIG_PPP=y
 CONFIG_PPP_BSDCOMP=y
 CONFIG_PPP_DEFLATE=y
@@ -315,6 +304,11 @@ CONFIG_INPUT_UINPUT=y
 # CONFIG_LEGACY_PTYS is not set
 CONFIG_HW_RANDOM=y
 CONFIG_GPIO_SYSFS=y
+CONFIG_THERMAL=y
+CONFIG_CPU_THERMAL=y
+CONFIG_CPUCORE_THERMAL=y
+CONFIG_GPU_THERMAL=y
+CONFIG_GPUCORE_THERMAL=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_MEDIA_USB_SUPPORT=y
 CONFIG_USB_VIDEO_CLASS=y
@@ -371,6 +365,7 @@ CONFIG_UDF_FS=y
 CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
 CONFIG_EXFAT_FS=y
+CONFIG_NTFS_FS=y
 CONFIG_TMPFS=y
 CONFIG_HFS_FS=y
 CONFIG_HFSPLUS_FS=y
@@ -381,7 +376,6 @@ CONFIG_PSTORE_RAM=y
 CONFIG_CIFS=y
 CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ISO8859_1=y
-CONFIG_NLS_UTF8=y
 CONFIG_PRINTK_TIME=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_READABLE_ASM=y
@@ -401,5 +395,3 @@ CONFIG_CRYPTO_PCBC=y
 CONFIG_CRYPTO_TWOFISH=y
 CONFIG_CRC_T10DIF=y
 CONFIG_CRC7=y
-CONFIG_NTFS_FS=y
-
diff --git a/arch/arm/configs/meson8b_tee_defconfig b/arch/arm/configs/meson8b_tee_defconfig
old mode 100755
new mode 100644
index 5ca61e4ff94a..1a051d6ae49b
--- a/arch/arm/configs/meson8b_tee_defconfig
+++ b/arch/arm/configs/meson8b_tee_defconfig
@@ -213,29 +213,14 @@ CONFIG_RICOH_PMU=y
 CONFIG_SECURITYKEY=y
 CONFIG_EFUSE=y
 CONFIG_AML_PLATFORM_THERMAL=y
-CONFIG_AML_VIRTUAL_THERMAL=y
-CONFIG_THERMAL=y
-CONFIG_CPU_THERMAL=y
-CONFIG_GPU_THERMAL=y
-CONFIG_CPUCORE_THERMAL=y
-CONFIG_GPUCORE_THERMAL=y
 CONFIG_AMLOGIC_USB=y
 CONFIG_MMC_AML=y
 CONFIG_AMLOGIC_SPI_NOR=y
 CONFIG_AML_NEXT_GEN_NAND=y
 CONFIG_AML_NAND_KEY=y
 # CONFIG_AM_NAND is not set
-#new maclogic
-CONFIG_NET_VENDOR_STMICRO=y
-CONFIG_STMMAC_ETH=y
-CONFIG_STMMAC_PLATFORM=y
-CONFIG_DWMAC_MESON=y
-CONFIG_STMMAC_DEBUG_FS=y
-CONFIG_STMMAC_DA=y
-CONFIG_NET_VENDOR_WIZNET=y
-CONFIG_AM_PHY=y
-#new maclogic
 CONFIG_AM_ETHERNET=y
+CONFIG_AM_PHY=y
 CONFIG_AML_PHY=y
 CONFIG_AML_LAN8720=y
 CONFIG_AML_IP101_PHY=y
@@ -243,8 +228,8 @@ CONFIG_AML_KSZ8091=y
 CONFIG_AML_RTL8211F=y
 CONFIG_AM_WIFI=y
 CONFIG_AM_WIFI_SD_MMC=y
-CONFIG_AM_WIFI_USB=y
 CONFIG_DHD_USE_STATIC_BUF=y
+CONFIG_AM_WIFI_USB=y
 CONFIG_SDIO_DHD_CDC_WIFI_40181_MODULE=m
 CONFIG_RTL8188EU=m
 CONFIG_RTL8189ES=m
@@ -297,6 +282,10 @@ CONFIG_DM_CRYPT=y
 CONFIG_DM_UEVENT=y
 CONFIG_NETDEVICES=y
 CONFIG_TUN=y
+CONFIG_STMMAC_ETH=y
+CONFIG_DWMAC_MESON=y
+CONFIG_STMMAC_DEBUG_FS=y
+CONFIG_STMMAC_DA=y
 CONFIG_PPP=y
 CONFIG_PPP_BSDCOMP=y
 CONFIG_PPP_DEFLATE=y
@@ -316,6 +305,11 @@ CONFIG_INPUT_UINPUT=y
 # CONFIG_LEGACY_PTYS is not set
 CONFIG_HW_RANDOM=y
 CONFIG_GPIO_SYSFS=y
+CONFIG_THERMAL=y
+CONFIG_CPU_THERMAL=y
+CONFIG_CPUCORE_THERMAL=y
+CONFIG_GPU_THERMAL=y
+CONFIG_GPUCORE_THERMAL=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_MEDIA_USB_SUPPORT=y
 CONFIG_USB_VIDEO_CLASS=y
@@ -373,6 +367,7 @@ CONFIG_UDF_FS=y
 CONFIG_MSDOS_FS=y
 CONFIG_VFAT_FS=y
 CONFIG_EXFAT_FS=y
+CONFIG_NTFS_FS=y
 CONFIG_TMPFS=y
 CONFIG_HFS_FS=y
 CONFIG_HFSPLUS_FS=y
@@ -383,7 +378,6 @@ CONFIG_PSTORE_RAM=y
 CONFIG_CIFS=y
 CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ISO8859_1=y
-CONFIG_NLS_UTF8=y
 CONFIG_PRINTK_TIME=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_READABLE_ASM=y
@@ -403,5 +397,3 @@ CONFIG_CRYPTO_PCBC=y
 CONFIG_CRYPTO_TWOFISH=y
 CONFIG_CRC_T10DIF=y
 CONFIG_CRC7=y
-CONFIG_NTFS_FS=y
-
-- 
2.19.0

